api/adaptive_load/adaptive_load.proto

```diff
@@ -0,0 +1,68 @@
+syntax = "proto3";
```

Comment:
Since this introduces a few proto files, can we add a file level comment to each one of them indicating (summarizing) what it contains?

Reply:
Done.